tree/dataframe/inc/ROOT/RResultHandle.hxx

```diff
@@ -0,0 +1,112 @@
+// Author: Stefan Wunsch, Enrico Guiraud CERN  09/2020
+
+/*************************************************************************
+ * Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_RDF_RRESULTHANDLE
+#define ROOT_RDF_RRESULTHANDLE
+
+#include "ROOT/RResultPtr.hxx"
+#include "ROOT/RDF/RLoopManager.hxx"
+#include "ROOT/RDF/RActionBase.hxx"
+#include "ROOT/RDF/Utils.hxx" // TypeID2TypeName
+
+#include <memory>
+#include <exception> // std::runtime_error
```

Comment:
several includes are missing, e.g. `type_info`, `vector`, `stringstream`  (although you could just use `std::string`), `stdexcept`...

Reply:
Done!

Reply:
In a separate commit ;)